src/config.ts

```diff
@@ -129,6 +149,14 @@ const GraphQLServer = (): string => {
   return `${LocalServerUrl}/graphql`;
 };
 
+const RemoteGraphQLServer = (): string => {
+  return `${RemoteHeadlessUrl}/graphql`;
+}
+
+const RemoteHeadlessUrl = (): string => {
+  return `${get("RemoteGraphQLServerHost")}:${get("RemoteGraphQLServerPort")}`;
```

Comment:
Could we use somthing like `RemoteGraphQLServerURL` instead of specifiying parts of it? It seems like it's always used as URL form.